test/helpers/index.js

```diff
@@ -0,0 +1,106 @@
+var path    = require('path'),
+  util      = require('util'),
+  Datastar  = require('../../lib'),
```

Comment:
Note: in `datastar-test-tools`, this referenced a completely different version of Datastar. Now, the integration tests will run against the code in the version under test.